docs/core/diagnostics/eventpipe.md

```diff
@@ -72,7 +72,7 @@ However, you can use the following environment variables to set up an EventPipe
 
 * `COMPlus_EventPipeOutputPath`: The path to the output EventPipe trace file when it's configured to run via `COMPlus_EnableEventPipe`. The default value is `trace.nettrace`, which will be created in the same directory that the app is running from.
 
-* `COMPlus_CircularBufferMB`: The size of the internal buffer that is used by EventPipe when it's configured to run via `COMPlus_EnableEventPipe`.
+* `COMPlus_CircularBufferMB`: The size of the internal buffer that is used by EventPipe when it's configured to run via `COMPlus_EnableEventPipe`, in megabytes. The default value is 1024.
```

Comment:
```suggestion
* `COMPlus_CircularBufferMB`: The size of the internal buffer that is used by EventPipe when it's configured to run via `COMPlus_EnableEventPipe`, in megabytes. The default value is 1024MB.
```

Reply:
Technically it is "1024" because the config expects the unit to already be in MB and expects the env var to be a numeric value.

Reply:
Perhaps, but it depends on your perception of what value means.  The most reacent noun was `internal buffer size`, os that would be the mostr logical reference, but I understand your perspective and I partially agree with it.

Reply:
Perhaps it would be clearest if we said the `default buffer size is 1024MB`

Reply:
Ah, I see what you mean. Let me reword this whole thing a little bit...

Reply:
Also I though coreclr interpreted all these as hex values.  So the default value is `400` -> `0x400` -> `1024` -> `1024MB`

Reply:
It's actually parsed as a string value.

Reply:
As a base 10 integer?

Reply:
Actually, you're right. This is registered as a dword value. Time for another update...